lxd/cluster/heartbeat.go

```diff
@@ -141,7 +217,8 @@ func Heartbeat(gateway *Gateway, cluster *db.Cluster) (task.Func, task.Schedule)
 				// We found a new node
 				nodes = append(nodes, currentNode)
 				heartbeatsWg.Add(1)
-				go sendHeartbeat(currentNode.ID, currentNode.Address, false)
+				heartbeatData = getHeartbeatRequest(true, raftNodes, nodes)
+				go sendHeartbeat(currentNode.ID, currentNode.Address, false, heartbeatData)
```

Comment:
If two new nodes appeared, this will result in the first of the two receiving a partial node list.

Reply:
@stgraber good spot, thanks.